kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Environment.kt

```diff
@@ -34,21 +34,29 @@ public fun env(name: String): String? {
             val lines = dotenvFile.readLines()
 
             for (line in lines) {
-                if (!line.contains('=')) {
+                var effectiveLine = line.trimStart()
+                if (effectiveLine.startsWith("#"))
+                    continue
```

Comment:
`MultiLineIfElse - [] at /home/runner/work/kord-extensions/kord-extensions/kord-extensions/src/main/kotlin/com/kotlindiscord/kord/extensions/utils/_Environment.kt:39:21`

tl;dr this needs braces